### DISCUSSION TITLE
How can I copy files to host?

### DISCUSSION THREAD

**Discussion by ccmywish** [uraimo/run-on-arch-action#137] [Q&A] [answered]
I'm using this great action to build on aarch64. 

I build the executable in the qemu environment, how can I copy the build artifact out to the host?

I'm not familiar with `qemu`, so could you please offer some help?

**Comment by uraimo** [comment, marked as answer]
Hi, you should be able to use the upload-artifact action: https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts#configuring-a-custom-artifact-retention-period

If you want to collect all artifacts in one place you can move them once the build is done, like the advanced example does: https://github.com/uraimo/run-on-arch-action#advanced-example

**Reply by martin-g** [reply]
@ccmywish The idea is to mount a host folder in the container, build, copy all wanted files into the mounted folder, and once run-on-arch-action is done you can find all your artifacts in the host folder.

**Comment by ccmywish** [comment]
Very helpful, thank you very much @uraimo @martin-g 👍
